mne/time_frequency/tests/test_multitaper.py

```diff
@@ -37,17 +40,21 @@ def test_multitaper_psd():
 
     import nitime as ni
     n_times = 1000
-    x = np.random.randn(5, n_times)
+    n_channels = 5
+    data = np.random.randn(n_channels, n_times)
     sfreq = 500
-    assert_raises(ValueError, multitaper_psd, x, sfreq, normalization='foo')
+    info = create_info([str(i) for i in range(n_channels)], sfreq, 'eeg')
```

Comment:
ah right right, I think I was one of the people that wanted that feature implemented in the first place and here I am not using it :P